docs/source/pillars/information_governance.md

```diff
@@ -88,6 +88,25 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
   - Mandatory
 ```
 
+### Internal Audit
+
+_An independent evaluation process within the TRE organisation that assesses and improves its internal controls, risk management, and governance._
```

Comment:
Are we confident an internal audit can be independent?

Reply:
What do you think @machintim?